Intermittent Cypress failures in Prod

```
  1) Data Browser 2019
       Nationwide/Institution/LienStatus:
     AssertionError: Has valid download link.: expected false to be true
```

- Only fails on DB 2019
- Not reproducible from my machine

--- a/cypress/integration/data-browser/2017.spec.js
+++ b/cypress/integration/data-browser/2017.spec.js
@@ -43,8 +43,8 @@ describe('Data Browser 2017', function () {
 
       // Test validity of download link
       if(isProd(HOST) || isBeta(HOST)){
-        cy.get('.QueryButton:first').dataUrl().then(({ status }) => {
-          assert.isTrue(status, 'Has valid download link.')
+        cy.get('.QueryButton:first').dataUrl().then(({ status, statusCode, url }) => {
+          assert.isTrue(status, `\nURL: ${url}\nStatus Code: ${statusCode}\nHas valid download link`)
         })
       }
     })
@@ -77,8 +77,8 @@ describe('Data Browser 2017', function () {
 
       // Test validity of download link
       if(isProd(HOST) || isBeta(HOST)){
-        cy.get('.QueryButton:first').dataUrl().then(({ status }) => {
-          assert.isTrue(status, 'Has valid download link.')
+        cy.get('.QueryButton:first').dataUrl().then(({ status, statusCode, url }) => {
+          assert.isTrue(status, `\nURL: ${url}\nStatus Code: ${statusCode}\nHas valid download link`)
         })
       }
     })
@@ -118,8 +118,8 @@ describe('Data Browser 2017', function () {
 
       // Test validity of download link
       if(isProd(HOST) || isBeta(HOST)){
-        cy.get('.QueryButton:first').dataUrl().then(({ status }) => {
-          assert.isTrue(status, 'Has valid download link.')
+        cy.get('.QueryButton:first').dataUrl().then(({ status, statusCode, url }) => {
+          assert.isTrue(status, `\nURL: ${url}\nStatus Code: ${statusCode}\nHas valid download link`)
         })
       }
     })
@@ -160,8 +160,8 @@ describe('Data Browser 2017', function () {
 
       // Test validity of download link
       if(isProd(HOST) || isBeta(HOST)){
-        cy.get('.QueryButton:first').dataUrl().then(({ status }) => {
-          assert.isTrue(status, 'Has valid download link.')
+        cy.get('.QueryButton:first').dataUrl().then(({ status, statusCode, url }) => {
+          assert.isTrue(status, `\nURL: ${url}\nStatus Code: ${statusCode}\nHas valid download link`)
         })
       }
     })

--- a/cypress/integration/data-browser/2018.spec.js
+++ b/cypress/integration/data-browser/2018.spec.js
@@ -43,8 +43,8 @@ describe('Data Browser 2018', function () {
 
       // Test validity of download link
       if(isProd(HOST) || isBeta(HOST)){
-        cy.get('.QueryButton:first').dataUrl().then(({ status }) => {
-          assert.isTrue(status, 'Has valid download link.')
+        cy.get('.QueryButton:first').dataUrl().then(({ status, statusCode, url }) => {
+          assert.isTrue(status, `\nURL: ${url}\nStatus Code: ${statusCode}\nHas valid download link`)
         })
       }
     })
@@ -77,8 +77,8 @@ describe('Data Browser 2018', function () {
 
       // Test validity of download link
       if(isProd(HOST) || isBeta(HOST)){
-        cy.get('.QueryButton:first').dataUrl().then(({ status }) => {
-          assert.isTrue(status, 'Has valid download link.')
+        cy.get('.QueryButton:first').dataUrl().then(({ status, statusCode, url }) => {
+          assert.isTrue(status, `\nURL: ${url}\nStatus Code: ${statusCode}\nHas valid download link`)
         })
       }
     })
@@ -121,8 +121,8 @@ describe('Data Browser 2018', function () {
 
       // Test validity of download link
       if(isProd(HOST) || isBeta(HOST)){
-        cy.get('.QueryButton:first').dataUrl().then(({ status }) => {
-          assert.isTrue(status, 'Has valid download link.')
+        cy.get('.QueryButton:first').dataUrl().then(({ status, statusCode, url }) => {
+          assert.isTrue(status, `\nURL: ${url}\nStatus Code: ${statusCode}\nHas valid download link`)
         })
       }
     })
@@ -165,8 +165,8 @@ describe('Data Browser 2018', function () {
 
       // Test validity of download link
       if(isProd(HOST) || isBeta(HOST)){
-        cy.get('.QueryButton:first').dataUrl().then(({ status }) => {
-          assert.isTrue(status, 'Has valid download link.')
+        cy.get('.QueryButton:first').dataUrl().then(({ status, statusCode, url }) => {
+          assert.isTrue(status, `\nURL: ${url}\nStatus Code: ${statusCode}\nHas valid download link`)
         })
       }
     })

--- a/cypress/integration/data-browser/2019.spec.js
+++ b/cypress/integration/data-browser/2019.spec.js
@@ -43,8 +43,8 @@ describe('Data Browser 2019', function () {
   
       // Test validity of download link
       if(isProd(HOST) || isBeta(HOST)){
-        cy.get('.QueryButton:first').dataUrl().then(({ status }) => {
-          assert.isTrue(status, 'Has valid download link.')
+        cy.get('.QueryButton:first').dataUrl().then(({ status, statusCode, url }) => {
+          assert.isTrue(status, `\nURL: ${url}\nStatus Code: ${statusCode}\nHas valid download link`)
         })
       }
     })
@@ -77,8 +77,8 @@ describe('Data Browser 2019', function () {
   
       // Test validity of download link
       if(isProd(HOST) || isBeta(HOST)){
-        cy.get('.QueryButton:first').dataUrl().then(({ status }) => {
-          assert.isTrue(status, 'Has valid download link.')
+        cy.get('.QueryButton:first').dataUrl().then(({ status, statusCode, url }) => {
+          assert.isTrue(status, `\nURL: ${url}\nStatus Code: ${statusCode}\nHas valid download link`)
         })
       }
     })
@@ -120,8 +120,8 @@ describe('Data Browser 2019', function () {
   
       // Test validity of download link
       if(isProd(HOST) || isBeta(HOST)){
-        cy.get('.QueryButton:first').dataUrl().then(({ status }) => {
-          assert.isTrue(status, 'Has valid download link.')
+        cy.get('.QueryButton:first').dataUrl().then(({ status, statusCode, url }) => {
+          assert.isTrue(status, `\nURL: ${url}\nStatus Code: ${statusCode}\nHas valid download link`)
         })
       }
     })
@@ -163,8 +163,8 @@ describe('Data Browser 2019', function () {
   
       // Test validity of download link
       if(isProd(HOST) || isBeta(HOST)){
-        cy.get('.QueryButton:first').dataUrl().then(({ status }) => {
-          assert.isTrue(status, 'Has valid download link.')
+        cy.get('.QueryButton:first').dataUrl().then(({ status, statusCode, url }) => {
+          assert.isTrue(status, `\nURL: ${url}\nStatus Code: ${statusCode}\nHas valid download link`)
         })
       }
     })

--- a/cypress/support/helpers.js
+++ b/cypress/support/helpers.js
@@ -25,7 +25,8 @@ export function urlExists(url) {
     const xhr = new XMLHttpRequest()
 
     xhr.onreadystatechange = function () {
-      if (xhr.readyState === 4) resolve({ url, status: xhr.status === 200 })
+      if (xhr.readyState === 4)
+        resolve({ url, status: xhr.status < 400, statusCode: xhr.status })
     }
 
     xhr.open('HEAD', url)

